code/src/components/FirstQue.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+const FirstQue = ({ FavoriteInput, FavoriteInputChange, Change }) => {
+  return (
+    <div class="container">
+      <h2> What's your favorite season? </h2>
+      <form className="form-container">
+        <label className="first-que">
+          <input
+            name="season"
+            type="radio"
+            value="Spring🌸🌸 "
+            onChange={() => FavoriteInputChange("Spring 🌸🌸")}
```

Comment:
Good that you managed to include emojiis, I experienced it to be quite difficult with react :)